cli_ftd/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ cmake_minimum_required(VERSION 3.17)
 # set(TARGET_BOARD "CC1352P_2_LAUNCHXL")
 set(TARGET_BOARD "CC1352R1_LAUNCHXL")
 
+set(UART_BAUD_RATE 9600) # 115200 (console) or 9600 (bluetooth)
```

Comment:
So is this the only thing that needs to be changed in order to toggle between Bluetooth and console?

Reply:
Clarified.